octobot_trading/api/exchange.py

```diff
@@ -32,3 +32,15 @@ def create_new_exchange(config, exchange_name,
                            is_collecting=is_collecting,
                            exchange_only=exchange_only,
                            backtesting_files=backtesting_files)
+
+
+def get_trading_pairs(exchange_manager):
```

Comment:
How can a module get the exchange_manager to call this API ?

Reply:
From ```create_new_exchange``` and ```get_exchange_manager_from_exchange_name``` that will be pushed with the next push :)

Reply:
Nice !